presentation/src/main/java/com/depromeet/presentation/home/HomeActivity.kt

```diff
@@ -31,84 +45,142 @@ class HomeActivity : BaseActivity<ActivityHomeBinding>(
+        clHomeThreeRecord.setOnClickListener { navigateToSeatRecordActivity() }
+        /** 시야후기 등록*/
+        clHomeRegisterSight.setOnClickListener { navigateToReviewActivity() }
+        clHomeNoRecord.setOnClickListener { navigateToReviewActivity() }
+        /** 시야 찾기 */
+        clMainFindSight.setOnClickListener { navigateToStadiumActivity() }
+
+        ibHomeSetting.setOnClickListener { /** 셋팅 이동 **/ }
+
+    }
+
+    private fun initObserver() {
+        viewModel.profile.asLiveData().observe(this) { state ->
             when (state) {
-                is UiState.Loading -> {
-                    toast("로딩중")
+                is UiState.Success -> {
+                    updateProfile(state.data)
                 }
 
-                is UiState.Empty -> {
-                    toast("빈값")
+                is UiState.Failure -> {
+                    toast("실패")
                 }
 
+                is UiState.Loading -> {}
+                is UiState.Empty -> {}
+            }
+        }
+        viewModel.recentReview.asLiveData().observe(this) { state ->
+            when (state) {
                 is UiState.Success -> {
-                    updateUi(state.data)
+                    updateRecentReview(state.data)
                 }
 
                 is UiState.Failure -> {
-                    toast("통신 실패")
+                    toast("실패")
                 }
+
+                is UiState.Loading -> {}
+                is UiState.Empty -> {}
             }
         }
-
-        viewModel.getInformation()
     }
 
-    
-    private fun updateUi(state: HomeUiState) {
-        with(binding) {
-            val profile = state.profile
-            val recentSight = state.recentSight
-
-            "Lv.${profile.level} ${profile.title}".also { tvHomeLevel.text = it }
-            setSpannableString(profile.nickName, profile.writeCount)
-            ivHomeCheerTeam.load(profile.cheerTeam)
-            tvHomeRecentRecordName.text = recentSight.location
-            tvHomeRecentRecordDate.text = recentSight.date
-
+    private fun updateRecentReview(data: RecentReviewResponse) = with(binding) {
+        setSpannableString(viewModel.nickname.value, viewModel.reviewCount.value)
+        tvHomeRecentRecordName.text = data.review.stadiumName
+        tvHomeRecentRecordDate.text = CalendarUtil.getFormattedDate(data.review.baseReview.dateTime)
 
-            sightList.forEachIndexed { index, view ->
-                view.visibility =
-                    if (index == recentSight.photoList.size) View.VISIBLE else View.GONE
+        sightList.forEachIndexed { index, view ->
+            view.visibility =
+                if (index == data.review.baseReview.images.size) View.VISIBLE else View.GONE
+        }
+        when (data.review.baseReview.images.size) {
+            0 -> {
+                tvHomeRecentRecordDate.visibility = View.GONE
+                tvHomeRecentRecordName.visibility = View.GONE
```

Comment:
```suggestion
                tvHomeRecentRecordName.visibility = GONE
```
요렇게 적으면 훨씬 깔끔하다고 생각하는데 어떠신가요 ?!

Reply:
와 여태 안드로이드하면서 처음안 사실이야.. 전체적으로 수정하두고 올리고 커밋 내역 남겨놓을게!